tests/blink2.cpp

```diff
@@ -0,0 +1,27 @@
+#include "mtrain.hpp"
+
+// Tests digital out using onboard LEDs in a looping pattern
+int main(void)
+{
+    DigitalOut led1 (LED1);
+    DigitalOut led2 (LED2);
+    DigitalOut led3 (LED3);
+    DigitalOut led4 (LED4);
+  
+  DigitalOut leds[] = {led1, led2, led3, led4};
```

Comment:
You can initialize the array at the same time you define the DigitalOut: 

`DigitalOut leds[] = { {LED1}, {LED2}, {LED3}, {LED4} };`